timescale-docker/init/restore.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+PGDATA=/home/postgres/pgdata/data
+
+pg_ctl -D $PGDATA stop -mf
+
+echo "*** Start restore ***"
+
+pgbackrest restore --stanza=poddb --delta --log-level-console=detail
+
+echo "*** Done restore ***"
+
+cp /etc/pg_hba.conf /home/postgres/pgdata/data/pg_hba.conf
+cp /etc/postgresql.conf /home/postgres/pgdata/data/postgresql.conf
+
+pg_ctl -D $PGDATA start -o '--archive-command=/bin/false'
+
+# sleep 30
+while true; do
+    MAX_BACKUP_WAL="$(pgbackrest info --output=json | python3 -c "import json,sys;obj=json.load(sys.stdin); print(obj[0]['archive'][0]['max']);")"
+    echo "Testing whether WAL file ${MAX_BACKUP_WAL} has been restored ..."
+    [ -f "${PGDATA}/pg_wal/${MAX_BACKUP_WAL}" ] && break
+    sleep 10;
+done
+
+pg_ctl -D $PGDATA stop -mf
+
+sleep 10
+
+pg_ctl -D $PGDATA start
+
+sleep 25
+
+psql -U postgres -d postgres -c "UPDATE pg_authid SET rolpassword = 'SCRAM-SHA-256\$4096:NK71fpsjRyAUUxSvFiDlGg==\$sBwig8n1Srb90HuK+jGb+hSWoQk9jVbvKLTX8mG/EIE=:2URiLDrB9cqgzyjKv8gt+LKwIuQ55nSm6S6wX8RgN20=' WHERE rolname='postgres';"
```

Comment:
Are these passwords meant to be here? 

Where do they come from? How do I change them.